apps/client/src/externals.ts

```diff
@@ -17,11 +17,11 @@ export const buyMeACoffeeUrl = 'https://buymeacoffee.com/cpvalente';
 
 // resolve environment
 export const appVersion = version;
-export const isProduction = import.meta.env.MODE === 'production';
-export const isDev = !isProduction;
+export const isDocker = import.meta.env.IS_DOCKER;
+export const isProduction = import.meta.env.PROD;
+export const isDev = import.meta.env.DEV;
```

Comment:
I believe this is not equivalent logic, the env can contain other values. before we just said that anything that is not the production build was dev (eg: tests)

Not sure if the difference causes any issues in our logic